### PR TITLE
[SDK-233] Add non hashing mode for key1-key10 - 2 README 

### PR DESCRIPTION
**Pull request by igor-alekseev-dev**
Pull Request Checklist
======================

Please read this for our PR Culture:
------------------------------------
https://incountry.atlassian.net/wiki/spaces/ED/pages/46858869/Pull+requests+culture+Draft

Work relates to:
----------------
Link(s) to Jira Task/User Story/Bug/Epic:
- [SDK-233](https://incountry.atlassian.net/browse/SDK-233)

Code Quality (Before submitting PR)
------------
- [x] Ensure code compiles and passes Sonar Lint.
- [x] Do your best to ensure integration tests pass. If they don't then...
- [x] Remove, ignore, or mark pending any brittle tests.
- [x] Remove commented out code.

Target Release
--------------
- [ ] Please put the target release vehicle this needs to be bundled with.
- [ ] Did you increment the version in package.json? What is the new version?

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -249,6 +250,14 @@ v3.0.0 release introduced a series of new fields available for storage. Below is
 ##### String fields, hashed:
 ```typescript
 recordKey
+profileKey
+serviceKey1
+serviceKey2
+```
+##### String fields, hashed if Storage options "hashSearchKeys" is set to true (by default it is):
+**WARNING** Also if `hashSearchKeys` is set to `false` this string fields has length limit 256
```
Comment (artemgolozubov) [inline README.md:258]:
```suggestion
**WARNING** If `hashSearchKeys` is set to `false` the next string fields will have length limitation of 256 chars at most
```

**Review by artemgolozubov — DISMISSED** [review]
Looks ok, but take a look at my comment please

File: README.md (hunk 1 above)
Comment (RommeroInCountry) [reply README.md:258]: the following string fields

**Review by RommeroInCountry — APPROVED** [review]
LGTM

**Review by artemgolozubov — APPROVED** [review]
LGTM

**Pull request merged**